sonar-core/src/main/java/org/sonar/core/qualityprofile/db/ActiveRuleDao.java

```diff
@@ -31,7 +31,8 @@
  * @deprecated use the ActiveRuleDao class defined in sonar-server
  */
 @Deprecated
-public class ActiveRuleDao implements ServerComponent {
+@ServerSide
+public class ActiveRuleDao {
```

Comment:
![INFO](https://raw.githubusercontent.com/henryju/image-hosting/master/info.png)Do not forget to remove this deprecated code someday.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)